test/tail.js

```diff
@@ -18,20 +18,50 @@ function writeLines(fd, count) {
   fs.closeSync(fd);
 }
 
+function endsWith(str, suffix) {
+  return str.indexOf(suffix, str.length - suffix.length) !== -1;
+}
+
 describe('tail', () => {
   temp.track();
 
   it('calls event line if new line appear in file', (done) => {
     temp.open(TEMP_FILE_PROFIX, (err, info) => {
       tail(info.path).on('line', (line) => {
-        line.should.equal('line0');
+        line.should.equal(`${info.path} - line0`);
```

Comment:
Here the new format of the output lines. Does it seems good to you ?
Maybe we should add the prefix only when there are multiple file tailed

Reply:
Fixed